automation_tools/satellite6/upgrade/__init__.py

```diff
@@ -146,10 +148,21 @@ def satellite6_capsule_upgrade(admin_password=None):
         print('Wrong Capsule Version Provided to upgrade to. '
               'Provide one of 6.1, 6.2')
         sys.exit(1)
+    if not os.environ.get('CAPSULE_SUBSCRIPTION') and not os.environ.get(
+            'CAPSULE_AK'):
+        print('One of Env. Var CAPSULE_SUBSCRIPTION or CAPSULE_AK should be'
```

Comment:
The if uses `and` operator and the message says `or`, what is the real intention here?